vcf2fhir/fhir_helper.py

```diff
@@ -1,598 +1,534 @@
-import os

```

Comment:
why does it shows whole file is changed ? did you change the formatting ?

Reply:
I think when I resolved the conflict, somehow I made it look like the whole file was the change.

Should I do the whole process again ??

Reply:
Yes its hard to review otherwise. You can revert back to commit where it was all good and start making changes from there.